### ISSUE TITLE
Accessibility 9.4.1.3 - status alerts are not detectable for screenreaders

### ISSUE THREAD

**Issue by hassmal** [Dataport/terminfinder-frontend#307]
# Bug

Wird eine Umfrage pausiert und der Abstimmungslink geöffnet erscheint eine Meldung („Die Umfrage kann derzeit nicht bearbeitet werden, da sie pausiert ist.“). Diese wird nicht für den Screenreader direkt ausgegeben.

Wird eine Umfrage bearbeitet erscheint ein Hinweis („Achtung…“), der auch nicht wiedergegeben wird.

## SOLL-Zustand

Um die Hinweise auch für Screenreader-Nutzer gleich deutlich zu machen, sollte die Meldung jeweils mit einem role=“alert“-Attribut oder alternativ mit „aria-live=“assertive“ versehen werden, so dass der Screenreader sie direkt ausgibt beim Betreten der Seite.

------------

# Bug

If a poll is paused and the voting link is opened, a message appears (“The poll cannot currently be edited because it is paused.”). This is not displayed directly for the screen reader.

If a poll is being edited, a message (“Attention...”) appears, which is also not displayed.

## TARGET state

In order to make the information immediately clear for screen reader users, the message should be provided with a role=“alert” attribute or alternatively with “aria-live=”assertive”, so that the screen reader displays it directly when entering the page.
